packages/charts/src/lib/Introduction.mdx

```diff
@@ -19,6 +19,5 @@ This packaage primarily provides a wrapper around the [Observable Plot](https://
 Note that to achieve the correct styling you should:
 
 - import `Plot` from `@ldn-viz/charts` (rather than `@observablehq/plot`
-)
```

Comment:
Why has this closing bracket gone? It should probably be moved to the previous line rather than deleted.